src/tests/meetings.test.js

```diff
@@ -0,0 +1,194 @@
+const request = require('supertest');
+const server = require('../server');
+const Meetings = require('../database/helpers/meetings');
+
+const MEETING_API_URL = '/api/meetings';
+
+afterEach(async () => {
+    await Meetings.truncate();
+})
+
+beforeEach(async () => {
+    await Meetings.truncate();
+})
+
+describe('MEETINGS ROUTER', () => {
+    describe('GET ROUTE /MEETINGS', () => {
+        it('should return status 200 on success', async () => {
+            const res = await request(server).get(MEETING_API_URL);
+            expect(res.status).toEqual(200);
+        });
+
+    })
+
+    describe('POST ROUTE /MEETINGS', () => {
+        it('should return status 201 on success', async () => {
+            const res = await request(server)
+            .post(MEETING_API_URL)
+            .send({
+                match_id: 1,
```

Comment:
before you do it, you *may* need to make sure this match exists

Reply:
should i use a helper function to getMatches? or do i do something different ?